server/database.env

```diff
@@ -4,4 +4,8 @@ password=
 database=
 DATABASE_URL=
 JWT_SECRET=eyJhbGciOiJIUzI1NiJ9.eyJSb2xlIjoiQWRtaW4iLCJJc3N1ZXIiOiJJc3N1ZXIiLCJVc2VybmFtZSI6IkphdmFJblVzZSIsImV4cCI6MTY3NDEyNjIwOSwiaWF0IjoxNjc0MTI2MjA5fQ.2vxaSmmfQ14Ol4Bgy3Cp62N5HbHBAnqlAkBDd
-JWT_EXPIRE=10
\ No newline at end of file
+JWT_EXPIRE=10
+GMAIL_CLIENT_ID=770124443966-6do9f24ebp54m.apps.googleusercontent.com
```

Comment:
SECRETS ??!

Reply:
It's not the real one, don't worry. It's for Unit Testing ;)